CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.12)
-project(ompl VERSION 1.6.0 LANGUAGES CXX)
-set(OMPL_ABI_VERSION 17)
+project(ompl VERSION 1.7.0 LANGUAGES CXX)
```

Comment:
Also bump the version in [py-bindings/setup.py](https://github.com/ompl/ompl/blob/d3f2cc76de0dfc33d1e6322adb2376b90e1c1241/py-bindings/setup.py#L153)